ucf-vertical-slide.php

```diff
@@ -11,3 +11,68 @@
 if ( ! defined( 'WPINC' ) ) {
     die;
 }
+
+define( 'UCF_VERTICAL_SLIDE__PLUGIN_URL', plugins_url( basename( dirname( __FILE__ ) ) ) );
+define( 'UCF_VERTICAL_SLIDE__STATIC_URL', UCF_VERTICAL_SLIDE__PLUGIN_URL . '/static' );
+define( 'UCF_VERTICAL_SLIDE__PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
+define( 'UCF_VERTICAL_SLIDE__FILEPATH', __FILE__ );
+
+// adding the custom post type path
+include_once 'includes/ucf-vertical-slider-posttype.php';
+
+// Shortcode function
+
+if ( ! function_exists( 'ucf_vertical_slide_shortcode' ) ) {
+
+	function ucf_vertical_slide_shortcode( $atts ) {
+		ob_start();
+		$defaults = array(
+			'slug' => 'default_slug',
+			'id' => 'default_id'
```

Comment:
It's very likely we do not want default values on these. If the user adds a `vertical_side` shortcode without a slug or id, what behavior do we want? We probably want to return out an empty string or else throw an error (depending on if we want it to fail gracefully or loudly)/

Reply:
Just removed the default value. Picked the gracefully method :D